lib/view_component/compiler.rb

```diff
@@ -21,6 +21,7 @@ def compiled?
     end
 
     def compile(raise_errors: false, force: false)
+      raise TemplateError.new(template_errors) if compiled? && raise_errors && template_errors.any?
```

Comment:
I didn't have time to dive into this, but this was required to get the test suite passing in cases where there are `template_errors`.

I _think_ there might have been a change around how/when templates are compiled in the test environment, since the component looked to be pre-compiled, even though I was running a single test in isolation.

Reply:
The templates are now `compiled? => true`, where they used to be `compiled? => false`, fwiw.